### ISSUE TITLE
Question about the training strategy.

### ISSUE THREAD

**Issue by MinghaoYe** [FlagOpen/RoboBrain#20]
Thanks for your valuable work.

This paper provides details on model architecture, training strategy and hyperparameters. As a beginner, I'm not quite clear about the underlying reasons behind these settings. When I want to train a model, I often don't know how to design my training strategy. I would like to know how you determine this settings. Are there any methodologies or experiences to share in this regard? Specifically, my questions are:
1. How are the trainable parameters determined in each stage?
2. How are the hyperparameters determined in each stage, including batch size, learning rate, and epoch, etc.?
3. How are the different parts of the model architecture selected, and are there any underlying reasons?

Thank you again for your excellent work. Looking forward to your reply.

**Comment by Yuheng2000** [comment]
Hi! Thank you for your thoughtful question.  

I believe the best way to start is to “stand on the shoulders of giants.” You can refer to the excellent technical reports released by teams such as Qwen and the LLaVA community, which provide valuable insights into model architecture design and training configurations.  

Beyond that, the key is to **experiment extensively** — try different training strategies, model architectures, and hyperparameter settings to find what works best for your data and objectives.  

In our case, we also built upon the strong foundation of **LLaVA-OneVision**, and conducted **broad explorations** on our own datasets to determine the optimal settings — including the choice of **LLM**, **ViT**, **hyperparameters**, and **data mixing ratios**.  

I hope this experience can be helpful to you!
